R/Model-class.R

```diff
@@ -2653,6 +2653,92 @@ OneParExpNormalPrior <- function(skel_probs,
   )
 }
 
+# OneParExpExpPrior ----
+
+## class ----
+
+#' `OneParExpExpPrior`
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' [`OneParExpExpPrior`] is the class for a standard CRM with an exponential prior on
+#' the log power parameter for the skeleton prior probabilities. It is an
```

Comment:
Hm I think the description can be very similar / analogous to the other `OneParExpNormalPrior` class

Reply:
> Hm I think the description can be very similar / analogous to the other `OneParExpNormalPrior` class

Actually it is also misleading in `OneParExpNormalPrior`...

Reply:
I think @wwojciech is right. It's misleading (its was copied and pasted from `OneParExpNormalPrior`) and should be corrected to
```
.... is the class for a standard CRM with an exponential prior on
power parameter for the skeleton prior probabilities.
```
because no exp transformation for the power parameter is used here.

Reply:
ok then let's fix both and make it consistent, thanks @0liver0815

Reply:
With regard to this, I think the names for these classes are wrong. It should be: `OneParLogNormalPrior` (instead of `OneParExpNormalPrior`) and `OneParExpPrior` (instead of `OneParExpExpPrior`).

Reply:
> 2. The `OneParExpExpPrior` is wrong, it should `OneParExpPrior`, as the `theta` parameter in this model follows `Exp` distr.

But there are other 1-parameter models. The part `OneParExp` should highlight that it is the 1 parameter power model and the second part which prior is used `ExpPrior.`

> 1. The `OneParExpNormalPrior` class name is a bit confusing. Better would be: `OneParLogNormalPrior`, as the `theta` parameter in this model follows `LogNormal` distr. I mean, the current naming is still acceptable, because `exp(X) ~ LogNormal`, when `X ~ Normal`, but I just think `OneParLogNormalPrior` would be better.

I agree. It is just that I remember that I see often in literature that the normal distribution is mentioned for the power parameter and then a truncation or exp transformation is used to make it positive. I guess therefore the nomenclature was `OneParExpNormalPrior`.

Reply:
> > 2. The `OneParExpExpPrior` is wrong, it should `OneParExpPrior`, as the `theta` parameter in this model follows `Exp` distr.
> 
> But there are other 1-parameter models. The part `OneParExp` should highlight that it is the 1 parameter power model and the second part which prior is used `ExpPrior.`
> 
> > 1. The `OneParExpNormalPrior` class name is a bit confusing. Better would be: `OneParLogNormalPrior`, as the `theta` parameter in this model follows `LogNormal` distr. I mean, the current naming is still acceptable, because `exp(X) ~ LogNormal`, when `X ~ Normal`, but I just think `OneParLogNormalPrior` would be better.
> 
> I agree. It is just that I remember that I see often in literature that the normal distribution is mentioned for the power parameter and then a truncation or exp transformation is used to make it positive. I guess therefore the nomenclature was `OneParExpNormalPrior`.

I see, thanks @0liver0815. Then `OneParExpNormalPrior` should be `OneParExpLogNormalPrior` or `OneParExpExpNormalPrior`, and `OneParExpExpPrior` is OK.

Reply:
hm I think no need to be too precise here. Let's stay with reasonable name choices even if they are not mathematically 100% precise

Reply:
Agree, but at least I think we should be consistent, because at the moment `Exp` denotes exponential transformation (for `OneParExpNormalPrior`) and at the same time, the `Exp` denotes the distribution for `theta` (for `OneParExpExpPrior `).

Reply:
@0liver0815 After discussion with @danielinteractive, we agreed on this:
(old model): OneParLogNormalPrior: f(x) ^ exp(theta), theta ~ Normal()
(new model): OneParExpPrior: f(x) ^ theta, theta ~ Exp()

Reply:
@wwojciech fine with me. I will rename the class accordingly.

Reply:
Thanks @0liver0815 - just your class. I will replace the old one with a separate issue.